scripts/drupal-vm/post-provision.php

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env php
 <?php
-
+// phpcs:ignoreFile
```

Comment:
Has to be ignored for now because PHPCS gets really confused when a shell file uses PHP